boards/phytec/phycore_am62x/doc/index.rst

```diff
@@ -40,6 +40,8 @@ features:
 +-----------+------------+-------------------------------------+
 | UART      | on-chip    | serial port                         |
 +-----------+------------+-------------------------------------+
+| Mailbox   | on-chip    | IPC Mailbox                         |
++-----------+------------+-------------------------------------+
```

Comment:
@vaishnavachath Just figured out you added this only to the A53 core and not for the phyboard-lyra M4. I would appreciate if you could update the M4 documentation as well.

Reply:
Added for both Phyboard Lyra M4 and A53